packages/test-cases/cases/cli/build-cmd/wasm/typescript/001-sanity/polywrap.build.yaml

```diff
@@ -0,0 +1,3 @@
+format: 0.3.0
+config:
+  scriptFile: ./wrap.js
```

Comment:
This ideally shouldn't be needed, it feels superfluous to always have to have a `polywrap.build.yaml` file IMO.

Reply:
Agreed. Removed in favor of using `polywrap.yaml`'s module file path